misc_scripts/change_toolchain_version.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python3
+"""
+change_toolchain_version: change the toolchain version in the GitHub Actions
+workflow configurations.
+"""
+
+# isort: STDLIB
+import argparse
+
+KEY_LSRT = r"# LOWEST SUPPORTED RUST TOOLCHAIN"
+KEY_CDRT = r"# CURRENT DEVELOPMENT RUST TOOLCHAIN"
+TOOLCHAIN_CHOICES = ["lowest", "current"]
+
+
+def search_file(new_version, search_key, old_verstring, new_verstring, filename):
+    """
+    Read the file.
+    """
+    with open(filename, "r") as file:
+        for line in file:
+
+            if search_key in line:
+                templine = line.replace(old_verstring, new_verstring)
+                if new_version not in templine:
```

Comment:
Instead of checking if ```new_version``` in ```templine```, just check if ```new_verstring``` is in ```templine```. The check works just as well, and that's one less argument to pass. Which is good, because probably next step is to figure out how to add the output file to the arguments.

Reply:
OK; for some reason, I was afraid that would never fail the check, but it appears to properly raise the error for an old version not found.

Reply:
So, this won't always be modifying the file it reads?

Reply:
I just added the output file parameter; it seems to work properly.